vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/rfc7519/jwt/JsonWebTokenHeader.kt

```diff
@@ -0,0 +1,7 @@
+package at.asitplus.wallet.lib.data.rfc7519.jwt
+
+import kotlinx.serialization.json.JsonObject
+import kotlin.jvm.JvmInline
+
+@JvmInline
+value class JsonWebTokenHeader(val jsonObject: JsonObject)
```

Comment:
This class is never used, and can't we use `JwsHeader` or `JweHeader` from signum?

Reply:
removed